cmd/trade.go

```diff
@@ -64,6 +62,13 @@ func init() {
 		}
 		log.Println(startupMessage)
 
+		if *logPrefix != "" {
```

Comment:
this should be the first thing we do, even before writing the starting message (above). This way the startup message, which includes the version and git hash, will also be a part of the log file.